app/src/main/java/in/testpress/retrofit_from_scratch/PostActivity.java

```diff
@@ -0,0 +1,60 @@
+package in.testpress.retrofit_from_scratch;
+
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.webkit.WebView;
+import android.widget.TextView;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class PostActivity extends AppCompatActivity {
+
+    public TextView textView;
+    public WebView webView;
+    String timeAgo;
+    String slug;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        setContentView(R.layout.activity_post);
+        Post post = getIntent().getParcelableExtra("Post");
```

Comment:
You can create a `public static final String` variable for post key, [refer here](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2213-string-constants-naming-and-values)